share/functions/__fish_complete_proc.fish

```diff
@@ -17,16 +17,21 @@ function __fish_complete_proc --description 'Complete by list of running process
 		set sed_cmds $sed_cmds 's/ .*//'
 		
 		# Erases weird stuff Linux gives like kworker/0:0
-		set sed_cmds $sed_cmds 's|/[0-9]:[0-9]]$||g'
+		set sed_cmds $sed_cmds '/.:./d'
 		
 		# Retain the last path component only
-		set sed_cmds $sed_cmds 's|.*/||g'
+		set sed_cmds $sed_cmds 's/.*\///'
 		
-		# Strip off square brackets. Cute, huh?
-		set sed_cmds $sed_cmds 's/[][]//g'
+		# Strip off  brackets. Cute, huh?
+		set sed_cmds $sed_cmds '/^\[/d'
+        set sed_cmds $sed_cmds '/\]$/d'
```

Comment:
Odd change in indentation here and below. I wouldn't bother fixing it because this module doesn't conform to the style guide. Too, I'm going to blanket restyle all the fish scripts to conform to the style guide in the near future. This is just a FYI comment to the broader community that we really need to be more vigilant about changes that aren't consistent with the style guide.